netmiko/mikrotik/mikrotik_ssh.py

```diff
@@ -124,6 +101,21 @@ def set_base_prompt(
         self.base_prompt = prompt
         return self.base_prompt
 
+    def send_config_set(
+        self,
+        config_commands: Union[str, Sequence[str], Iterator[str], TextIO, None] = None,
+        exit_config_mode: bool = False,
+        enter_config_mode: bool = False,
+        **kwargs: Any
+    ) -> str:
+        """No config mode on MikroTik"""
+        return super().send_config_set(
+            config_commands=config_commands,
+            exit_config_mode=exit_config_mode,
+            enter_config_mode=enter_config_mode,
+            **kwargs
+        )
+
```

Comment:
This shouldn't be necessary...the `NoConfig` class should automatically make the default `send_config_set()` parent method work (without changing any of the send_config_set arguments).